src/utils/dataFormat.go

```diff
@@ -25,7 +25,6 @@ var (
 	K = math.Pow(10, 3)
 	M = math.Pow(10, 6)
 	G = math.Pow(10, 9)
-	T = math.Pow(10, 12)
 )
```

Comment:
@mihirkestur Although this makes sense, maybe it would be better to retain T (Trillion, Peta Bytes), maybe making Peta Bytes as the last case would be better. I know it's unlikely to have metrics in pB, but it is a possibility.

Reply:
Alright sure!